{
	"id": "CVE-2021-23418",
	"vulnerabilities": [
		{
			"content": {
				"id": "CVE-2021-23418",
				"description": "The package glances before 3.2.1 are vulnerable to XML External Entity (XXE) Injection via the use of Fault to parse untrusted XML data, which is known to be vulnerable to XML attacks.",
				"severity": [
					{
						"type": "cvss_v2",
						"source": "nvd@nist.gov",
						"cvss_v2": {
							"vector": "AV:N/AC:L/Au:N/C:P/I:P/A:P",
							"base_score": 7.5,
							"nvd_base_severity": "HIGH",
							"temporal_score": 7.5,
							"nvd_temporal_severity": "HIGH",
							"environmental_score": 7.5,
							"nvd_environmental_severity": "HIGH"
						}
					},
					{
						"type": "cvss_v31",
						"source": "nvd@nist.gov",
						"cvss_v31": {
							"vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H",
							"base_score": 9.8,
							"base_severity": "CRITICAL",
							"temporal_score": 9.8,
							"temporal_severity": "CRITICAL",
							"environmental_score": 9.8,
							"environmental_severity": "CRITICAL"
						}
					},
					{
						"type": "cvss_v31",
						"source": "report@snyk.io",
						"cvss_v31": {
							"vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:R/S:U/C:L/I:L/A:L",
							"base_score": 6.3,
							"base_severity": "MEDIUM",
							"temporal_score": 6.3,
							"temporal_severity": "MEDIUM",
							"environmental_score": 6.3,
							"environmental_severity": "MEDIUM"
						}
					}
				],
				"cwe": [
					{
						"source": "nvd@nist.gov",
						"cwe": [
							"CWE-611"
						]
					}
				],
				"references": [
					{
						"source": "nvd.nist.gov",
						"url": "https://nvd.nist.gov/vuln/detail/CVE-2021-23418"
					},
					{
						"source": "report@snyk.io",
						"url": "https://github.com/nicolargo/glances/commit/4b87e979afdc06d98ed1b48da31e69eaa3a9fb94"
					},
					{
						"source": "report@snyk.io",
						"url": "https://github.com/nicolargo/glances/commit/85d5a6b4af31fcf785d5a61086cbbd166b40b07a"
					},
					{
						"source": "report@snyk.io",
						"url": "https://github.com/nicolargo/glances/commit/9d6051be4a42f692392049fdbfc85d5dfa458b32"
					},
					{
						"source": "report@snyk.io",
						"url": "https://github.com/nicolargo/glances/issues/1025"
					},
					{
						"source": "report@snyk.io",
						"url": "https://snyk.io/vuln/SNYK-PYTHON-GLANCES-1311807"
					}
				],
				"published": "2021-07-29T18:15:07.727Z",
				"modified": "2021-08-05T18:58:05.28Z"
			},
			"segments": [
				{
					"ecosystem": "cpe"
				}
			]
		}
	],
	"detections": [
		{
			"ecosystem": "cpe",
			"conditions": [
				{
					"criteria": {
						"operator": "OR",
						"criterias": [
							{
								"operator": "OR",
								"criterias": [
									{
										"operator": "OR",
										"criterias": [
											{
												"operator": "OR",
												"criterions": [
													{
														"vulnerable": true,
														"fix_status": {
															"class": "unknown"
														},
														"package": {
															"cpe": "cpe:2.3:a:glances_project:glances:*:*:*:*:*:*:*:*"
														},
														"affected": {
															"type": "semver",
															"range": [
																{
																	"lt": "3.2.1"
																}
															]
														}
													}
												]
											}
										]
									}
								]
							}
						]
					}
				}
			]
		}
	],
	"data_source": {
		"id": "nvd-api-cve",
		"raws": [
			"vuls-data-raw-nvd-api-cve/2021/CVE-2021-23418.json"
		]
	}
}
